zips/draft-pacu-regtest-local-consensus-definition.md

```diff
@@ -0,0 +1,314 @@
+```
+  ZIP: Unassigned
+  Title: Regtest: Definition of a Local Consensus test mode for Zcash Full-Nodes
+  Owners: ZIP Editors
```

Comment:
Owners must be specific named people who have each explicitly consented to be an Owner. I haven't, so this can't just be replaced by the current ZIP Editors (nor do the ZIP Editors have time to be assigned the role of maintaining ZIPs by others). Normally the Owners of the ZIP are the people who wrote it.